neuralprophet/conformal.py

```diff
@@ -158,35 +159,60 @@ def _get_q_hat(self, df_cal: pd.DataFrame, noncon_scores: np.ndarray) -> float:
 
         return q_hat
 
-    def plot(self, plotting_backend: str):
+    def plot(self, plotting_backend: str = "default"):
         """Apply a given conformal prediction technique to get the uncertainty prediction intervals (or q-hats).
 
         Parameters
         ----------
             plotting_backend : str
                 specifies the plotting backend for the nonconformity scores plot, if any
 
-                Options
-                    * ``matplotlib``: Use matplotlib backend for plotting
-                    * ``plotly``: Use the plotly backend for plotting
+                                Options
+                * ``plotly-resample``: Use the plotly backend for plotting in resample mode. This mode uses the
```

Comment:
plotly-resample with the comment that you might not get any error message.
plotly-auto